server.ts

```diff
@@ -0,0 +1,139 @@
+import express from "express";
+import cors from "cors";
+import { useState } from "react";
+
+const app = express();
+
+// add json handling
+app.use(express.json());
+
+app.use(cors())
+
+const initialBoard = ['', '', '', '', '', '', '', '', '']
+
+
+let games: Game[] = [{
```

Comment:
Interesting that you have **games** as an array, which requires you to then create a findGame function.

You could also treat **games** as a dictionary, ie an object of the format...

games = {
["id123": {
id: "id123",
board: etc
etc


And that way later on you can get a game by just going to **games[id]**

Reply:
to be clear, in the example it would be...

games["id123"]

which would give you the game object that looks like...

{
id: "id123",
board: etc
etc